### ISSUE TITLE
/lib/anbox/swiftshader is not represented in anbox snap edge

### ISSUE THREAD

**Issue by tts-035** [anbox/anbox#1989]
**Please describe your problem:**
There are no swiftshader libraries in the edge channel 
Because of this anbox does not work in software graphics rendering mode

**What were you expecting?:**
Anbox to start in software graphics rendering mode

**Additional info:**

**Comment by pupergay** [comment]
Yes, put another video card on your computer, I think the dollar will be extra
